schema.sql

```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE DATABASE vet_clinic;
+
+CREATE TABLE animals(
+  id INT,
+  name TEXT,
```

Comment:
[OPTIONAL] You can mark the id column as primary key 👌

Reply:
Thanks for the suggestion.